webapp/cypress/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "types": ["cypress", "@testing-library/cypress"],
+    "baseUrl": ".",
```

Comment:
'baseUrl' and 'paths' are already in the extended config

Reply:
this one overwrite the base config, it's intended